src/commcare_cloud/ansible/roles/postgresql/tasks/main.yml

```diff
@@ -43,7 +43,7 @@
     path: '{{ postgresql_dir_path }}'
     owner: postgres
     group: postgres
-    mode: 0755
```

Comment:
I think this will need to be deployed everywhere?

Reply:
This will already be the case everywhere since PG won't start if the permissions are wrong.

Reply:
I don't believe that's true as the standbys currently have `755`:

```
TASK [postgresql : Make new postgresql directory] ******************************
ok: [10.247.164.25]
--- before
+++ after
@@ -1,4 +1,4 @@
 {
-    "mode": "0755",
+    "mode": "0700",
     "path": "/opt/data/postgresql"
 }

changed: [10.247.164.84]
--- before
+++ after
@@ -1,4 +1,4 @@
 {
-    "mode": "0755",
+    "mode": "0700",
     "path": "/opt/data/postgresql"
 }

changed: [10.247.164.24]
```

Reply:
Interesting, in the past when I've tried starting PG with 0755 I get this:
```
FATAL: data directory "/path/to/cluster" has group or world access
DETAIL: Permissions should be u=rwx (0700).
```

Either way I don't think there's harm in changing it to `0700`